package.json

```diff
@@ -1,14 +1,16 @@
 {
   "name": "qwant-maps-fonts",
   "version": "1.0.0",
+  "repository": "github:QwantResearch/fonts",
   "authors": [
     "Petr Sloup <petr.sloup@klokantech.com>",
     "Qwant"
   ],
   "scripts": {
-    "build": "node ./generate"
+    "build": "node ./generate",
+    "prepare": "npm run build"
   },
-  "dependencies": {
+  "devDependencies": {
```

Comment:
Why this change?

Reply:
the packages published to npm will provide the dist files. Rebuilding them at install time is unnecessary and should never so I added the source ttf fonts to .npmignore, and `dependencies -> devDependencies`

Reply:
this result in less dependencies installed at install time